setup.py

```diff
@@ -1,29 +1,20 @@
 from setuptools import find_packages, setup
 
 DISTNAME = 'xskillscore'
-VERSION = '0.0.15'
```

Comment:
the point of setuptools is to have this line removed. area you ok with this @raybellwaves ?